diff -Naur uavs3d-7b1dd734c926a23f8134b42dbfaac4d22cd6ba1d.old/CMakeLists.txt uavs3d-7b1dd734c926a23f8134b42dbfaac4d22cd6ba1d/CMakeLists.txt
--- uavs3d-7b1dd734c926a23f8134b42dbfaac4d22cd6ba1d.old/CMakeLists.txt	2022-03-16 09:42:43.512772295 +0100
+++ uavs3d-7b1dd734c926a23f8134b42dbfaac4d22cd6ba1d/CMakeLists.txt	2022-03-16 10:25:05.054515426 +0100
@@ -15,4 +15,3 @@
     target_link_libraries(uavs3dec m)
 endif()
 target_link_libraries(uavs3dec uavs3d)
-#target_link_libraries(uavs3dec dl)
diff -Naur uavs3d-7b1dd734c926a23f8134b42dbfaac4d22cd6ba1d.old/source/CMakeLists.txt uavs3d-7b1dd734c926a23f8134b42dbfaac4d22cd6ba1d/source/CMakeLists.txt
--- uavs3d-7b1dd734c926a23f8134b42dbfaac4d22cd6ba1d.old/source/CMakeLists.txt	2022-03-16 09:42:43.523772547 +0100
+++ uavs3d-7b1dd734c926a23f8134b42dbfaac4d22cd6ba1d/source/CMakeLists.txt	2022-03-16 10:03:33.023983234 +0100
@@ -128,6 +128,8 @@
 MESSAGE(STATUS "BUILD_SHARED_LIBS \t\t: false")
 endif()
 add_library(${LIBNAME} ${DIR_UAVS3D_SRC} ${UAVS3D_ASM_FILES})
+set_target_properties(${LIBNAME} PROPERTIES VERSION 1.2)
+set_target_properties(${LIBNAME} PROPERTIES SOVERSION 1)
 
 if (NOT MSVC)
     target_link_libraries(${LIBNAME} m)
